shared/src/jsMain/kotlin/com/github/jetbrains/rssreader/core/RssReader.kt

```diff
@@ -0,0 +1,26 @@
+package com.github.jetbrains.rssreader.core
+
+import com.github.jetbrains.rssreader.core.datasource.network.FeedLoader
+import com.github.jetbrains.rssreader.core.datasource.storage.FeedStorage
+import com.russhwolf.settings.JsSettings
+import io.github.aakira.napier.DebugAntilog
+import io.github.aakira.napier.Napier
+import kotlinx.serialization.json.Json
+
+@ExperimentalJsExport
```

Comment:
Possibly it is redundant because it is not marked with `@JsExport`?

Reply:
yes it was